runtime/common/src/apis.rs

```diff
@@ -271,23 +271,20 @@ macro_rules! impl_runtime_apis_plus_common {
 					let is_transactional = false;
 					let validate = true;
 
+					// Estimated encoded transaction size must be based on the heaviest transaction
+					// type (EIP1559Transaction) to be compatible with all transaction types.
 					let mut estimated_transaction_len = data.len() +
-						// to: 20
-						// from: 20
-						// value: 32
-						// gas_limit: 32
+						// chain_id 8 bytes
 						// nonce: 32
-						// 1 byte transaction action variant
-						// chain id 8 bytes
+						// max_priority_fee_per_gas: 32
+						// max_fee_per_gas: 32
+						// gas_limit: 32
+						// action: 21 (enum varianrt + call address)
+						// value: 32
+						// access_list: 1 (empty vec size)
 						// 65 bytes signature
-						210;
+						255;
```

Comment:
Also, it seems like we lost `from`. Is this because there is no such thing in an actual transaction (it's recovered from its signature)...?

Reply:
`210 + 64 - 20 => 254`, still off by one :laughing:

Reply:
> it seems like we lost from. Is this because there is no such thing in an actual transaction

Yes exactly

>  still off by one

The one come from access_list, even if the access liost is empty, the scale encoded format take 1 byte for the empty Vec.